docs/cr/application.giantswarm.io_v1alpha1_chart.yaml

```
@@ -16,6 +16,7 @@ spec:
       name: f2def-chart-values
       namespace: f2def
       resourceVersion: ""
+    skipCrdInstall: false
```

```suggestion
  install:
    skipCRDs: false
```

@yasn77 So far we've done these type of config extensions by adding annotations like for skipping forced upgrades. But I think we should start extending the CRD which is fine as long as we don't make breaking changes.

I compared this with the helmrelease CRD as I was interested to see how Flux do this. I like how they structure this. As its aligned with the helm option.  WDYT?

https://toolkit.fluxcd.io/components/helm/helmreleases/

Also I think we need to make both install and skipCRDs optional. As in most cases we don't need to set this option.

---

Yeh I like adding `Install.SkipCRDs`, it gives us an option of adding additional install time options such as skipping hooks (similar to fluxcd approach)